plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/io/FileDownloader.java

```diff
@@ -86,8 +85,8 @@ public IPath download(URL url, IProgressMonitor monitor) throws IOException {
     connection.setReadTimeout(DEFAULT_READ_TIMEOUT_MS);
     connection.setRequestProperty("User-Agent", CloudToolsInfo.USER_AGENT);
     try (InputStream inputStream = new BufferedInputStream(connection.getInputStream());
-         OutputStream outputStream =
-             new BufferedOutputStream(new FileOutputStream(downloadedFile))) {
+         OutputStream outputStream = 
+             new BufferedOutputStream(Files.newOutputStream(downloadedFile.toPath()))) {
```

Comment:
I think the original code has some minor problem. I read somewhere that these streams should not be chained in a try-with-resources block to ensure closing in case it fails in the middle of the chain. That is, it should be

```java
try (...
    OutputStream out = new Files.newOutputStream(...);
    BufferedOutputStream bufferedOut = new BufferedOutputStream(out)) {
```